### PR TITLE
WIP: added an emoji to the test file

### PR DESCRIPTION
**Pull request by jacobwilliams**

### CODE REVIEW

**Comment by jacobwilliams** [comment]
☹️

**Comment by zbeekman** [comment]
It's not actually JSON-Fortran that's failing.. it's the json validation. I recommend a few changes to .travis.yml to help debug. First of all switch the PPA providing CMake to `george-edison55-precise-backports` from `kalakris-cmake` and set a global environment variable (in travis-ci settings or in .travis.yml, the latter is probably a better approach): `CTEST_OUTPUT_ON_FAILURE=TRUE`. The PPA switch will bring in a newer CMake (that I know supports the environment variable). Good luck!

**Comment by jacobwilliams** [comment]
Thanks, I'll try it.

File: CMakeLists.txt [hunk 1]
```diff
@@ -267,7 +267,7 @@ if ( ENABLE_TESTS )
       get_filename_component ( TESTNAME "${VALID_JSON}" NAME )
       add_test ( NAME validate-${TESTNAME}
 	WORKING_DIRECTORY "${DATA_DIR}/inputs"
-	COMMAND ${JSONLINT} "--allow=nonescape-characters" "${VALID_JSON}" )
+	COMMAND ${JSONLINT} "--allow=nonescape-characters --allow non-portable" "${VALID_JSON}" )
```
Comment (zbeekman) [inline CMakeLists.txt]: It looks like this change may be causing additional failures... Maybe it is a problem with the file encoding? Are emoji like the one you included utf8 or utf16? I doubt the Fortran ISO 10646 support is actually supporting utf16, heck it probably isn't even really supporting utf8. (I am under the impression that some/most compilers store individual characters in 4 bytes--for memory alignment reasons--and it's possible that they are still doing this with ISO10646, b/c, IIRC, this is just ucs4... but I could be wrong)

It looks like jsonlint will take -e to specify explicitly the input/output encoding codec. You could try `-e utf8` or `-e utf16` to resolve the problem. Also, running the test on a local computer after `pip install --upgrade git+https://github.com/dmeranda/demjson.git && jsonlint --version` should give you equivalent results.

**Pull request closed without merging**